views/partials/login.ejs

```diff
@@ -13,5 +13,5 @@
 </form>
 
 <div class="container">
-  <button id="signup-action" class="btn btn-warning">Sign up</button>
+  <button id="logout-action" class="btn btn-warning">Sign up</button>
```

Comment:
not that much I think, it should be ok to move them to chinese in minutes i think. Or I might start use chinese in the future...

I think we should only hire people who know english to be the Admin. LOL ...

Ready to merge